container/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+LABEL type=application/vnd.module.wasm.content.layer.v1+wasm
```

Comment:
This dockerfile feels wrong. 

Usually it should be two stages, one building the wasm binary and the other only bearing the binary with `scratch` base image.